<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyqItem</class>
 <widget class="QWidget" name="PyqItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>252</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>487</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>487</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyqItem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
background-color: #ffffff;</string>
  </property>
  <widget class="QLabel" name="m_avatarLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../res/res.qrc">:/img/avatar.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_nicknameLabel">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>5</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #576b95;</string>
   </property>
   <property name="text">
    <string>花未眠</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="m_contentTextBrowser">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>21</y>
     <width>402</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>402</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>402</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;这是测试朋友圈，这是测试朋友圈，这是测试朋友圈&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="m_imageWidget" native="true">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>55</y>
     <width>399</width>
     <height>133</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_timeLabel">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>195</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #9e9e9e</string>
   </property>
   <property name="text">
    <string>30分钟前</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_commentPushButton">
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>195</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>评论</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_likePushButton">
   <property name="geometry">
    <rect>
     <x>387</x>
     <y>195</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>点赞</string>
   </property>
  </widget>
  <widget class="QWidget" name="m_commentWidget" native="true">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>212</y>
     <width>399</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTextBrowser" name="m_likeListTextBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>399</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;测试用户1、测试用户2、测试用户3点赞了&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="m_commentInnerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>22</y>
      <width>399</width>
      <height>0</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="m_deletePushButton">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>195</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
